pkg/controller/directvolumemigration/rsync.go

```diff
@@ -565,6 +576,65 @@ func (t *Task) getRsyncRoute(namespace string) (string, error) {
 	return route.Spec.Host, nil
 }
 
+func (t *Task) createRsyncPassword(destClient compat.Client) error {
+	var letters = []byte("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+	random.Seed(time.Now().UnixNano())
+	password := make([]byte, 6)
+	for i := range password {
+		password[i] = letters[random.Intn(len(letters))]
+	}
+
+	secret := corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: migapi.OpenshiftMigrationNamespace,
+			Name:      "rsync-pass",
+			Labels: map[string]string{
+				"app": "directvolumemigration-rsync-transfer",
+			},
+		},
+		Data: map[string][]byte{
+			"password": password,
+		},
+	}
+	err := destClient.Create(context.TODO(), &secret)
```

Comment:
we probably want this in the controller cluster and not in the destination cluster. You can do `t.Client.Create(...)` instead

Reply:
Yup thanks. I made this same mistake when I was first only testing with destination cluster as controller cluster. Good catch.